spec/rails_helper.rb

```diff
@@ -70,6 +76,7 @@
 
   # Include helpers
   config.include JsonSpec::Helpers
+  config.include AuthenticatedRequests, type: :feature
   config.include AuthenticatedRequests, type: :request
```

Comment:
```suggestion
  [:request, :feature].each do |type|
    config.include AuthenticatedRequests, type: type
  end
```

Reply:
I rearranged the `config.include` calls to group the three related to authentication and do something like this.

Reply:
👍